api/api/views/audio_views.py

```diff
@@ -48,6 +53,50 @@ def get_queryset(self):
 
     # Extra actions
 
+    @creator_collection
+    @action(
+        detail=False,
+        methods=["get"],
+        url_path="source/(?P<source>[^/.]+)/creator/(?P<creator>.+)",
+    )
+    def creator_collection(self, request, source, creator):
+        """
+        Get a collection of audio items by a specific creator from the specified source.
```

Comment:
Why are these overridden in the audio/image views? Is this supposed to override the less specific docstring ("collection of media items...") from `creator_collection`? In the [docs](http://localhost:50280/v1/#tag/audio/operation/audio_source_creator) I still see the `media` verbiage, and I don't see anything different in the definitions here.

Reply:
I think the reason we might be missing the media-specific documentation in the docs is because [these schema extensions do not include a `desc` parameter](https://github.com//WordPress/openverse/blob/ae04e7ebf33b3107bbcb5b693f0ebde110590862/api/api/docs/image_docs.py#L149-L176), which the other endpoint docs do.